style/components/main-content.scss

```diff
@@ -15,8 +15,20 @@
     left: 0;
     bottom: 0;
     right: 0;
-    height: 40px;
-    padding: 5px;
+
+    .search {
+      @extend .card;
+      margin: 10px;
+      padding: 0 15px;
```

Comment:
Can we move the padding to `input`? If a user clicks on what appears to be the search bar, but to the left of the input, the input won't be selected.
It should be possible to make the whole bar selectable